python/modifier_relativistic.cpp

```diff
@@ -0,0 +1,66 @@
+//------------------------------------------------------------------------------
+// © 2021-2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+// clang-format off
+#include "module.hpp"
+
+template<typename T>
+void relativistic_helper(py::module_ & m) {
+  m.def("Relativistic", [](T eos, const Real cl){
+    return RelativisticEOS<T>(std::move(eos), cl);
+  }, py::arg("eos"), py::arg("cl"));
+}
+
+template<typename T>
+void relativistic_class_helper(py::module_ & m, std::string name) {
+  eos_class<RelativisticEOS<T>>(m, std::string("Relativistic") + name)
+    .def(
+      py::init<T, Real>(),
+      py::arg("eos"), py::arg("cl")
+    );
+}
+
+template<typename T>
+void relativistic_eos_class(py::module_ & m, const char * name) {
+  // define Relativistic utility function
```

Comment:
For developers we should document that these lists of template specializations need to be updated in tandem with the list in `eos.hpp`. Doesn't need to be now, but I wonder if we can automate that some how with template magic? @dholladay00 what are your thoughts?